src/operator-docs/combination/combineLatest.ts

```diff
@@ -1,34 +1,42 @@
-import { OperatorDoc } from "../operator.model";
+import { OperatorDoc } from '../operator.model';
 
 export const combineLatest: OperatorDoc = {
-  name: "combineLatest",
-  operatorType: "combination",
+  name: 'combineLatest',
+  operatorType: 'combination',
   signature:
-    "public combineLatest(observables: ...Observable, project: function): Observable",
+    'public combineLatest(observables: ...Observable, project: function): Observable',
   useInteractiveMarbles: true,
   parameters: [
     {
-      name: "other",
-      type: "Observable",
-      attribute: "",
+      name: 'other',
+      type: 'Observable',
+      attribute: '',
       description:
-        "An input Observable to combine with the source Observable. More than one input Observables may be given as argument."
+        'An input Observable to combine with the source Observable. More than one input Observables may be given as argument.'
     },
     {
-      name: "other",
-      type: "function",
-      attribute: "optional",
+      name: 'other',
+      type: 'function',
+      attribute: 'optional',
       description:
-        "An optional function to project the values from the combined latest values into a new value on the output Observable."
+        'An optional function to project the values from the combined latest values into a new value on the output Observable.'
     }
   ],
-  marbleUrl: "http://reactivex.io/rxjs/img/combineLatest.png",
+  marbleUrl: 'http://reactivex.io/rxjs/img/combineLatest.png',
   shortDescription: {
     description: `
       Combines multiple Observables to create an Observable whose values
       are calculated from the latest values of each of its input Observables.
     `,
-    extras: []
+    extras: [
+      {
+        type: 'Tip',
+        text: `
+          Note: combineLatest only start to emit when all sources have emitted at least once.
+          By adding a default start value to the sources with .startWith, it will activate right away.
```

Comment:
Small typo, `will only start` instead of `only start`. Also, since we are referencing `startWith` here can we make that a link? Thanks!

Reply:
For sure :) I'm onto it.

Reply:
Angular sanitizes the links inside the tips. Has this happened before anywhere?

Reply:
I've written a `safeHtml` pipe which returns `return this._sanitizer.bypassSecurityTrustHtml(html);`, equivalent to the existing `safeUrl` pipe. Is this an acceptable solution for you?